app/src/main/java/com/example/tasky/feature_agenda/presentation/Agenda.kt

```diff
@@ -164,6 +174,34 @@ fun AgendaContent(
                 )
                 Spacer(modifier = Modifier.padding(top = 10.dp))
                 HeaderSmallBold(title = headerDateText, modifier = Modifier.align(Alignment.Start))
+                Spacer(modifier = Modifier.padding(top = 10.dp))
+                LazyColumn(
+                    modifier = Modifier.fillMaxSize(),
+                    verticalArrangement = Arrangement.spacedBy(8.dp)
+                ) {
+                    if (agendaItems != null) {
+                        items(agendaItems) { agendaItem ->
+                            AgendaCard(
+                                title = agendaItem.title,
+                                details = agendaItem.details,
+                                date = when (agendaItem) {
+                                    is AgendaItem.Event -> formatTimeBasedOnEvent(
+                                        agendaItem.fromDate,
+                                        agendaItem.toDate
+                                    )
+
+                                    is AgendaItem.Task, is AgendaItem.Reminder ->
+                                        formatTimeBasedOnEvent(
+                                            agendaItem.date,
+                                            null
+                                        )
+                                },
+                                cardType = agendaItem.cardType,
+                                isChecked = false
+                            )
+                        }
```

Comment:
A quick and easy way to add animations when items are inserted/changed/reordered is to add unique keys to each lazy column item and by adding a simple modifier:
```
items(
    items = agendaItems,
    key = { it.id } // use a property you know is unique per item
) {
    AgendaCard(
        ...,
        modifier = Modifier.animateItem()
    )
}
```